### PR TITLE
Update memberlist pure proxy and refactor IPFS hashes

### PR DESCRIPTION
**Pull request by sophialittlejohn**
### Description

This pull request...

Closes #<issue-number>

### Approvals

- [x] Dev
- [ ] Dev
- [ ] Designer
- [ ] Product

### CODE REVIEW

**Review by offerijns — APPROVED** [review]
👍

File: centrifuge-app/.env-config/.env.catalyst [hunk 1]
```diff
@@ -14,4 +14,4 @@ REACT_APP_SUBSCAN_URL=
 REACT_APP_TINLAKE_NETWORK=goerli
 REACT_APP_INFURA_KEY=bf808e7d3d924fbeb74672d9341d0550
 REACT_APP_WHITELISTED_ACCOUNTS=
-REACT_APP_MEMBERLIST_ADMIN_PURE_PROXY=kALJqPUHFzDR2VkoQYWefPQyzjGzKznNny2smXGQpSf3aMw19
\ No newline at end of file
+REACT_APP_MEMBERLIST_ADMIN_PURE_PROXY="4bo2vNkwZtr2PuqppWwqya6dPC8MzxqZ4kgnAoTZyKo9Kxq8"
```
Comment (jpangelle) [inline centrifuge-app/.env-config/.env.catalyst:17]: Is this supposed to have quotes?

Comment (sophialittlejohn) [reply centrifuge-app/.env-config/.env.catalyst:17]: It shouldn't make a difference but I'll revert!

File: onboarding-api/env-vars/catalyst.env [hunk 2]
```diff
@@ -1,5 +1,5 @@
 REDIRECT_URL=https://app-catalyst.k-f.dev
-MEMBERLIST_ADMIN_PURE_PROXY=kALJqPUHFzDR2VkoQYWefPQyzjGzKznNny2smXGQpSf3aMw19
+MEMBERLIST_ADMIN_PURE_PROXY="4bo2vNkwZtr2PuqppWwqya6dPC8MzxqZ4kgnAoTZyKo9Kxq8"
```
Comment (jpangelle) [inline onboarding-api/env-vars/catalyst.env]: Same here

**Review by jpangelle — APPROVED** [review]

**Pull request merged**